### DISCUSSION TITLE
What is the rule on creating variants within a theme?

### DISCUSSION THREAD

**Discussion by bdielman** [dracula/dracula-theme#507] [Q&A] [answered]
I've had a request to include a version of the Dracula theme for Nova without italics. I know there are variants in other themes, like the 'soft' version of Dracula that's included with the VS Code theme. I have no issue with adding this option to users, but when trying to keep within the [spec](https://spec.draculatheme.com), what's the rule we should follow when requests like these come in or is it something that should be handle case by case?

**Comment by luxonauta** [comment, marked as answer]
The specs are general guidelines, not imperative; case-by-case is always better! 😉
